.circleci/config.yml

```diff
@@ -284,7 +284,8 @@ commands:
             TARBALL_PATH="/tmp/${CIRCLE_PROJECT_REPONAME}-$(uname)-standard-blst.tar.gz"
             RELEASE_NAME="${CIRCLE_PROJECT_REPONAME}-$(uname)-standard-blst"
 
-            ./scripts/build-release.sh filcrypto $(cat ./rust-toolchain) --verbose --locked --no-default-features --features blst
+            # Note: the standard blst release uses the portable configuration for maximum compatibility
+            ./scripts/build-release.sh filcrypto $(cat ./rust-toolchain) --verbose --locked --no-default-features --features blst --features blst-portable
```

Comment:
if we want to change the default, we need to do so for the pairing one too, as it uses `blst` now for signatures as well

Reply:
Ah, didn't realize, thanks!